### ISSUE TITLE
V24.5 PIT summary

### ISSUE THREAD

**Issue by manolo** [vaadin/platform#6634]
## Blockers
- [x] https://github.com/vaadin/hilla/issues/2825
- [x] https://github.com/vaadin/base-starter-spring-gradle/issues/231
- [x] https://github.com/vaadin/copilot-internal/issues/3382
- [x] https://github.com/vaadin/start/issues/3225
- [x] https://github.com/vaadin/flow/issues/20063
- [x] https://github.com/vaadin/flow/issues/20097
- [x] https://github.com/vaadin/control-center/issues/553
- [x] https://github.com/vaadin/flow/issues/19956 (the fix is in Hilla)
- [x] https://github.com/vaadin/collaboration-engine-internal/issues/1048
- [x] https://github.com/vaadin/docs/issues/3576
- [x] https://github.com/vaadin/collaboration-engine-internal/issues/1049
- [x] https://github.com/vaadin/ui-test-generator/issues/84
- [x] https://github.com/vaadin/flow/issues/19741
- [x] https://github.com/vaadin/skeleton-starter-kotlin-spring/issues/21
- [x] https://github.com/vaadin/flow/issues/19831
- [x] https://github.com/vaadin/start/issues/3159
- [x] https://github.com/vaadin/flow/issues/19870
- [x] https://github.com/vaadin/flow/issues/19876
- [x] https://github.com/vaadin/start/issues/3167
- [x] https://github.com/vaadin/flow/issues/19895
  - [x] https://github.com/vaadin/start/issues/3169
- [x] https://github.com/vaadin/web-components/issues/7789
- [x] https://github.com/vaadin/control-center/issues/515
- [x] https://github.com/vaadin/control-center/issues/516
- [x] https://github.com/vaadin/designer-tutorial/issues/267

## Non Blockers
- [x] https://github.com/vaadin/control-center/issues/632
- [x] https://github.com/vaadin/flow/issues/19874
- [x] https://github.com/vaadin/control-center/issues/518
- [ ] https://github.com/vaadin/collaboration-engine-internal/issues/1050

## GA Blocker
- [x] https://github.com/vaadin/control-center/issues/510
- [x] https://github.com/vaadin/platform/issues/6799
- [x] https://github.com/vaadin/flow/issues/20216

## After Stable release
- [x] https://github.com/vaadin/business-app-starter-flow/issues/380

**Comment by ZheSun88** [comment]
hackathon findings:
fixes before GA:
- [ ] https://github.com/vaadin/flow-components/issues/6691

**Comment by ZheSun88** [comment]
@knoobie yes, we are awaring this issue and it will be handled when we go to final.. (very soon. ;-) )
